addons/website_theme_install/static/src/less/website_theme_install.less

```diff
@@ -1,42 +1,164 @@
+@o-theme-kanban-gray: #fcfcfc;
+
 .o_kanban_view.o_theme_kanban {
+    background-color: @o-theme-kanban-gray;
+
+    /// Un-grouped Layout (default)
     &.o_kanban_ungrouped {
         .o-justify-content(center);
 
         .o_kanban_record {
-            .o-flex(0, 0, auto);
-            width: 100%;
-            @media (min-width: @screen-sm-min) {
-                width: 45%;
-            }
+            .o-flex(0, 1, auto);
+            width: 250px;
```

Comment:
@qsm-odoo .. mmm...
Good question.. actually I would like review the entire margin/padding/layout stuff here, a lot has changed from the first attempt and for the best 👍 

I will push another [REVIEW] comment on top